__tests__/app.test.js

```diff
@@ -27,6 +27,16 @@ describe("GET-/api/categories", () => {
         });
       });
   });
+  describe("404: Route not found", () => {

```

Comment:
Id suggest these are a bit unnecessary, assume you have 1 test for invalid path already which will cover this and the similar test for api/review. Maybe leave them in for now but don't feel the need to add them going forward.